app/models/piece.rb

```diff
@@ -93,4 +93,20 @@ def remains_on_board?(to_x:, to_y:)
       false
     end
   end
+
+  def successful_capture(to_x:, to_y:)
+    return false if obstructed_diagonally?(to_x:, to_y:)
+    return false if obstructed_vertically?(to_x:, to_y:)
+    return false if obstructed_horizontally?(to_x:, to_y:)
+    current_color = current_piece.color
+    target_piece = Piece.where(x_position: to_x, y_position: to_y, active: true, color: !current_color)  # Add requirement that target MUST be opponent's color
```

Comment:
Could say `target_piece = Piece.where(x_position: to_x, y_position: to_y, active: true)` and then later do something like 
```
if target_piece.color == color
  'piece is friend'
elsif target_piece.color != color
  'piece is foe'
end
```
Where the `color` is the current piece's color and  `target_piece.color` is the target piece's color